insights/specs/default.py

```diff
@@ -696,7 +697,7 @@ def cmd_and_pkg(broker):
         Attributes:
             COMMANDS (list): List of the specified commands that need to check the provider package.
         """
-        COMMANDS = ['java']
+        COMMANDS = ['java', 'httpd']
```

Comment:
As this change will also impact the existing but deprecated parser&combiner `PackageProvidesJava{All}`, please remove the spec `package_provides_java` of the deprecated parser.